python/Dockerfile

```diff
@@ -4,9 +4,9 @@ CMD bash
 ENV WORKING=/opt/gink/python
 RUN mkdir -p $WORKING
 WORKDIR $WORKING
-COPY requirements.txt ./requirements.txt
-RUN pip install --no-cache-dir -r requirements.txt
-COPY gink ./gink
+COPY . .
+RUN pip install .
+RUN pip install 'gink[test]'
+RUN pip install 'gink[lint]'
```

Comment:
I think you can replace the  ` RUN pip install .[test,lint]` 
Otherwise you'd be pulling down the existing gink[test] dependencies from what's already uploaded to pypi.